dcos_installer/upgrade.py

```diff
@@ -58,35 +49,44 @@
     fi
 fi
 
+SKIP_CHECKS=false
+
+if [[ $# -ne 0 ]]; then
```

Comment:
you can blend these into a single if statement.
`if [[ $# -ne 0 ]] && [[ "$1" = "skip-checks" ]]; then`